### DISCUSSION TITLE
ScriptReference/Physics.Raycast

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#366] [Notes]
# ScriptReference/Physics.Raycast

https://docs.unity3d.com/ScriptReference/Physics.Raycast.html

**Comment by docscomments** [comment]
**Comment by `@Maddieman`:**  

To ignore multiple layers easily you can use a [`LayerMask`](https://docs.unity3d.com/ScriptReference/LayerMask.html) and set its value in the inspector.

**Comment by vertxxyz** [comment]
See also: [`RaycastCommand`](https://docs.unity3d.com/ScriptReference/RaycastCommand.html) for efficiently performing bulk raycasts.
